ledgers/ccf/scripts/fetch_ledger_authority.py

```diff
@@ -82,10 +82,13 @@ def Main() :
     try :
         config = toml.load(options.ccf_config)
     except :
-        LOG.error('unable to load ccf configuration file {0}'.format(options.ccf_config))
-        sys.exit(-1)
+        config = None
+        #LOG.error('unable to load ccf configuration file {0}'.format(options.ccf_config))
+        #sys.exit(-1)
+
+    #network_cert = config["start"]["network-cert-file"]
+    network_cert = os.path.join(CCF_Keys, "networkcert.pem")
 
-    network_cert = config["start"]["network-cert-file"]
```

Comment:
will look at the full logic later... but this does not look like the right flow. you've changed the way we are doing things. maybe we should have a config file that we use for our deployment since we are moving away from the cchost.toml file.

Reply:
same answer as above. The final PR will not have this new flow, we will use cchost to start the network and use the configure.py to carry out all member steps. cchost API, config/governance templates all have changed, which I need to understand before I can move away from the "sandbox" model to "deployment model", Thanks.